packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -97,9 +97,9 @@ class SimControl {
     final Map<String, dynamic> devicesSection = _list(SimControlListSection.devices);
```

Comment:
I'd change the definition of `_list()` to return `Map<String, dynamic>` instead of a raw `dynamic.

Reply:
Done.